app/src/androidTest/java/org/mozilla/tv/firefox/helpers/MainActivityTestRule.kt

```diff
@@ -0,0 +1,64 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.tv.firefox.helpers
+
+import android.preference.PreferenceManager
+import android.support.test.InstrumentationRegistry
+import android.support.test.espresso.IdlingRegistry
+import android.support.test.rule.ActivityTestRule
+import org.mozilla.tv.firefox.MainActivity
+import org.mozilla.tv.firefox.onboarding.OnboardingActivity.Companion.ONBOARD_SHOWN_PREF
+import org.mozilla.tv.firefox.pocket.PocketOnboardingActivity.Companion.POCKET_ONBOARDING_SHOWN_PREF
+
+/**
+ * A [org.junit.Rule] to handle shared test set up for tests on [MainActivity].
+ *
+ * @param initialTouchMode true to enable "touch mode", false otherwise
+ * @param launchActivity true to automatically launch the activity, false otherwise
```

Comment:
Nit: this `launchActivity` description is unclear to me, but the description in `ActivityTestRule` is good.  Suggest either directing readers to it or poaching it entirely.